src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  padding: 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  color: $main-text-color;
+  font-family: 'Roboto', sans-serif;
+
+  &__block {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+
+    &--upper {
+      margin-bottom: 24px;
+      margin-top: 16px;
+    }
+    &--lower {
+      margin-top: 0px;
+      margin-bottom: 16px;
+    }
+  }
+
+  &__image {
+    margin-top: 16px;
+    margin-bottom: 40px;
+    margin-left: 3px;
```

Comment:
You can use shorthand properties instead
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax